we use gnu-only string functions, so get the prototypes to prevent
wrong implicit function decls

--- gcc-3.3.6/gcc/treelang/parse.y
+++ gcc-3.3.6/gcc/treelang/parse.y
@@ -51,6 +51,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#define __USE_GNU
 #include <string.h>
 
 #include "ansidecl.h"
